### ISSUE TITLE
#include <oneapi/mkl.hpp>

### ISSUE THREAD

**Issue by tb5874** [oneapi-src/oneMKL#438]
hello, i'm windows10, oneapi user.

when i write oneapi code with onemkl, i can't use oneapi::mkl::rng::device namespace.
cmakelists.txt include compile options
/Qmkl:sequential
/Qmkl-ilp64:sequential

https://github.com/oneapi-src/oneMKL/blob/develop/include/oneapi/mkl.hpp
https://github.com/oneapi-src/oneMKL/blob/develop/include/oneapi/mkl/rng.hpp
in there, header did not include 'rng/device.hpp'.

i just want to know reason.

**Comment by Rbiessy** [comment]
I'm not familiar with the rng domain but I can see that the [example](https://github.com/oneapi-src/oneMKL/blob/develop/examples/rng/device/uniform.cpp#L40) and [tests](https://github.com/oneapi-src/oneMKL/blob/develop/tests/unit_tests/rng/device/include/moments.hpp#L38) directly include this file as well. Maybe someone from @oneapi-src/onemkl-rng-write can confirm whether this is intentional?

**Comment by aelizaro** [comment]
@Rbiessy, yes, having a standalone rng/device.hpp is intentional, as we wanted to stress out what device API are a completely different entity and has header-based implementation. So the expectation is that users may independently include rng/device.hpp functionality within sycl kernels and don't need to link to any binaries.

**Comment by Rbiessy** [comment]
Closing as there are no issues, this is done by design.
